atom/browser/native_window_mac.mm

```diff
@@ -470,6 +470,15 @@ - (void)drawRect:(NSRect)dirtyRect {
   return [window_ styleMask] & NSFullScreenWindowMask;
 }
 
+void NativeWindowMac::SetBounds(const gfx::Rect& bounds) {
+  Move(bounds);
+}
+
+gfx::Rect NativeWindowMac::GetBounds() {
+  return gfx::Rect(GetPosition(),
```

Comment:
We can probably implement `GetBounds` with Cocoa, and then implement `GetPosition` and `GetSize` with `GetBounds` to make code more decent.